src/Unused/Types.hs

```diff
@@ -37,20 +43,56 @@ newtype DirectoryPrefix = DirectoryPrefix String deriving (Eq, Show, Ord)
 
 resultsFromMatches :: [TermMatch] -> TermResults
 resultsFromMatches m =
-    TermResults
-        { matches = m
+    calculateLikelihood $ TermResults
+        { resultTerm = resultTerm' terms
```

Comment:
also, since this is simply instantiation, you don't need to use record syntax if you don't want to. You could list the props like so `TermResults (resultTerm' terms) m totalFiles' totalOccurences' High`